spec/logic_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../lib/board'
+require_relative '../lib/logic'
+
+describe ChangeNumbers do
+  describe 'number_change:' do
+    it 'change an element from the array with a symbol' do
+      change = ChangeNumbers.new
+      expect(change.number_change(2, [1, 2, 3, 4, 5, 6, 7, 8, 9], 'X')).to eql([1, 'X', 3, 4, 5, 6, 7, 8, 9])
+    end
+  end
+end
+
+describe Analize do
+  describe 'conditions:' do
+    let(:change) { Analize.new }
+    let(:array) { ['X', 'O', 'X', 4, 5, 6, 7, 8, 9] }
+    let(:array2) { ['X', 2, 3, 'X', 5, 6, 'X', 8, 9] }
+    let(:array3) { %w[X O X X O X O X O] }
+    let(:player) { 'Juan' }
+    let(:sim) { 'X' }
+    let(:checkout) { [1] }
+
+    it 'gives the condition if it is a win in a row' do
+      expect(change.conditions(array, player, sim, checkout)).to eql(nil)
+    end
```

Comment:
- [ ] I think this `it` block could be changed to be a negative test case because no message should be displayed if there are no winning or draw combinations so instead of the description it has right now the description could be changed to a negative one for example:

```ruby
it "doesn't display a win or draw message"
  expect(change.conditions(array, player, sim, checkout)).to eql(nil)
end
.
.
```

Reply:
Great review, thanks